packages/features/bookings/Booker/Booker.tsx

```diff
@@ -59,6 +61,7 @@ const BookerComponent = ({
   // In Embed we give preference to embed configuration for the layout.If that's not set, we use the App configuration for the event layout
   // But if it's mobile view, there is only one layout supported which is 'mobile'
   const layout = isEmbed ? (isMobile ? "mobile" : validateLayout(embedUiConfig.layout) || _layout) : _layout;
+  const newExtraDays = useRef<number>(isTablet ? extraDaysConfig[layout].tablet : extraDaysConfig[layout].desktop);
```

Comment:
Would this constant conflict with extraDays declared later? Or is this the one we should this instead of the other one?
Can we find a better name for this? I'm not sure what it does by it's name.

Reply:
The reason it was created is to calculate the no of days in between the available slots that's currently visible on screen. 
<img width="277" alt="Screenshot 2023-07-14 at 8 57 46 AM" src="https://github.com/calcom/cal.com/assets/123581387/d419d9b1-c140-428e-8026-0d3883b2c50b">

And it's only used for ColumnView. So we can keep **ColumnViewExtraDays**